runtime-modules/content/src/lib.rs

```diff
@@ -1754,7 +1772,15 @@ decl_module! {
         }
 
         /// Start video nft open auction
-        #[weight = 10_000_000] // TODO: adjust weight
+        /// <weight>
+        ///
+        /// ## Weight
+        /// `O (W)` where:
+        /// - W : member whitelist length
+        /// - DB:
+        ///    - O(1)
```

Comment:
Note it will change to `O(W)` once https://github.com/Joystream/joystream/pull/4221 is merged